ignite/pkg/goanalysis/goanalysis.go

```diff
@@ -279,3 +279,59 @@ func createUnderscoreImport(imp string) *ast.ImportSpec {
 		},
 	}
 }
+
+// HasStructFieldsInPkg finds the struct into a package folder and checks
+// if the field exists inside the struct.
+func HasStructFieldsInPkg(pkgPath, structName string, fields []string) (bool, error) {
+	absPath, err := filepath.Abs(pkgPath)
+	if err != nil {
+		return false, err
+	}
+	fileSet := token.NewFileSet()
+	all, err := parser.ParseDir(fileSet, absPath, func(os.FileInfo) bool { return true }, parser.ParseComments)
```

Comment:
Shouldn't the filter be `nil`?

```suggestion
	all, err := parser.ParseDir(fileSet, absPath, nil, parser.ParseComments)
```